community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/store/BatchingNeoStores.java

```diff
@@ -350,14 +351,11 @@ public void close()
         flushAndForce();
 
         // Flush out all pending changes
-        propertyKeyRepository.close();
-        labelRepository.close();
-        relationshipTypeRepository.close();
+        safeClose( propertyKeyRepository, labelRepository, relationshipTypeRepository );
```

Comment:
this safeClose is not really safe, also we already have correct one in IOUtils, so please use that instead

Reply:
OK, yeah that's a nice util